microservice/src/test/java/de/gsi/microservice/concepts/majordomo/MajordomoProtocol.java

```diff
@@ -0,0 +1,418 @@
+package de.gsi.microservice.concepts.majordomo;
+
+import static org.zeromq.ZMQ.Socket;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+import org.zeromq.SocketType;
+import org.zeromq.ZFrame;
+import org.zeromq.ZMQ;
+import org.zeromq.ZMsg;
+import org.zeromq.util.ZData;
+
+import zmq.SocketBase;
+
+/**
+ * Majordomo Protocol (MDP) definitions and implementations according to https://rfc.zeromq.org/spec/7/
+ *
+ * @author rstein
+ */
+public class MajordomoProtocol {
```

Comment:
Codacy found an issue: [The utility class name 'MajordomoProtocol' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6307742)